README.rst

```diff
@@ -70,7 +70,10 @@ Usage
 
 #. (Optional) Sink Historical event data to ClickHouse::
 
-    tutor [dev|local] run lms ./manage.py lms transform_tracking_logs --source_provider LOCAL --source_config '{"key": "/openedx/data", "container": "logs", "prefix": "tracking.log"}' --transformer_type xapi
+    tutor [dev|local] do transform_tracking_logs \
+      --source_provider LOCAL --source_config '{"key": "/openedx/data", "container": "logs", "prefix": "tracking.log"}' \
+      --transformer_type xapi --deduplicate
+
```

Comment:
Can we add another note here about situations where you would need to deduplicate and why you shouldn't do that if you don't need to (takes a long time).